src/coreComponents/physicsSolvers/wavePropagation/integratedTests/test.xml

```diff
@@ -0,0 +1,153 @@
+<?xml version="1.0" ?>
+
+<Problem>
+  <Solvers>
+    <!-- define the solver -->
+    <!-- we'll have to discuss the cflFactor with Randy -->
+    <!-- define the source coordinates -->
+    <!-- define the time source frequency -->
+    <!-- define the receiver coordinates -->
```

Comment:
General comment for the XML files:

1. I suggest removing the comments that were in the original xml file (like `we'll have to discuss the cflFactor with Randy`) if there are no longer necessary.
2. I suggest renaming this integrated test (currently, `test.xml`) to a more meaningful name.
3. Just to make sure, do you need to have two benchmarks xml files (they look very similar)? If having two is useful, please keep both.

Once you have renamed `test.xml`, I will push the baselines to the `integratedTests` repository.